### PR TITLE
refactor FormField & input length

### PR DESCRIPTION
**Pull request by B1s9l**
# Pull Requests Checklist

Assignee:

- \[ \] code commented?
- \[ \] deactivated things not needed? (e.g. other features which only were prepared)
  - \[ \] alerts that this is todo?
- \[ \] test run localhost?
- \[ \] tested manually all features and edge cases?
- \[ \] documentation added / updated?
- \[ \] pre-commit checks passed?
- \[ \] pipelines passed?

Reviewer:

- \[ \] correct branch?
- \[ \] are there any code smells?
- \[ \] did pipeline pass?
- \[ \] is the code well documented?
- \[ \]tested manually all features and edge cases?

### CODE REVIEW

File: src/styles/ui/FormField.scss
Comment (Monti934) [inline src/styles/ui/FormField.scss:39]: image

Please set min/max height/width for textare

I also liked the old styling better, did you change it on purpose?

**Review by Monti934 — CHANGES_REQUESTED** [review]

File: src/styles/ui/FormField.scss
Comment (B1s9l) [reply src/styles/ui/FormField.scss:39]: 1. Doesn't happen for me, could you share more details so I can recreate this issue?
2. Since I'm quite advanced with most of my issues/tasks I'm planning to go over the styling the upcoming days. I'd like to make everything a bit more modern, but would obviously revert it to the old styling if you prefers so.

Comment (Monti934) [reply src/styles/ui/FormField.scss:39]: 1. the is a draghandle in the bottom right corner of the textarea with which you can change its size. Either set max/min or deactivate that if possible
2. you can change the styling but at the moment it looks very unfinished and especially comments looks very off

Comment (B1s9l) [reply src/styles/ui/FormField.scss]: 1. Done (also disabled some formfields when not in edit mode, was previously forgotten)
2. I understand the concern, I also get the part about the comments. for me it is important to set a visual boundary between the team information and the comments, would like to leave this as it is for now and do proper styling later. its easier to refactor the entire component now, before we're working on other stuff that might conflict again.

File: src/components/popups/InspectTask.tsx
Comment (Monti934) [inline src/components/popups/InspectTask.tsx:149]: It should only say "Edit Task" when we are in Edit Mode

**Review by Monti934 — CHANGES_REQUESTED** [review]

File: src/components/popups/InspectTask.tsx
Comment (B1s9l) [reply src/components/popups/InspectTask.tsx]: Has been fixed, thanks a lot for your help!

**Review by Monti934 — CHANGES_REQUESTED** [review]
image

I'm sorry, but I can't approve this if we go from the left state of the application to the right one. Maybe this is on me and we need to discuss this with the team

**Pull request closed without merging**
